whylog/constraints/verifier.py

```
@@ -182,6 +182,25 @@ def constraints_or(cls, clues_lists, effect, constraints, constraint_manager):
         return causes
 
     @classmethod
+    def constraints_not(cls, clues_lists, effect, constraints, constraint_manager):
+        """
+        provide investigation if there is zero or one constraint,
+        because only in such cases NOT linkage has sense
```

if something doesn't make sense, assert it instead of failing silently